mirar/pipelines/winter/models/_diff.py

```diff
@@ -66,14 +72,24 @@ def validate_savepath(cls, field_value: str):
         assert os.path.exists(field_value)
         return field_value
 
-    @validator("uexpid")
-    @classmethod
-    def validate_expid(cls, field_value: int):
+    def insert_entry(
+        self, returning_key_names: str | list[str] | None = None
+    ) -> pd.DataFrame:
         """
-        Ensure that expid exists in exposures table
+        Insert entry into database
 
-        :param field_value: field value
-        :return: field value
+        :param returning_key_names: names of keys to return
+        :return: dataframe of inserted entries
         """
-        assert Exposure._exists(keys="uexpid", values=field_value)
-        return field_value
+        dbconstraints = DBQueryConstraints()
+        dbconstraints.add_constraint(
+            column="stackid",
+            accepted_values=self.stackid,
+        )
+        _update_database_entry(
```

Comment:
Does this work if there are no matches? (Genuine question)

Reply:
Yes, seems to.